tools/angular-on-nginx/admin-nrced.bc.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${NAME}
+objects:
+  - kind: ImageStream
+    apiVersion: image.openshift.io/v1
+    metadata:
+      name: nrpti
+      labels:
+        app: ${GROUP_NAME}
+        build: ${NAME}-build
+    spec:
+      lookupPolicy:
+        local: false
+  - apiVersion: build.openshift.io/v1
+    kind: BuildConfig
+    metadata:
+      labels:
+        app: ${GROUP_NAME}
+        build: ${NAME}-build
+      name: ${NAME}-build
+    spec:
+      completionDeadlineSeconds: 300
+      failedBuildsHistoryLimit: 5
+      nodeSelector: null
+      output:
+        to:
+          kind: ImageStreamTag
+          name: nrpti:latest
+      resources:
+        limits:
+          cpu: "2"
+          memory: 4Gi
+        requests:
+          cpu: "1"
+          memory: 2Gi
+      runPolicy: Parallel
+      source:
+        dockerfile: |-
+          FROM angular-app:latest
+          COPY * /tmp/app/dist/
+          CMD  /usr/libexec/s2i/run
+        images:
+          - as: null
+            from:
+              kind: ImageStreamTag
+              name: ${SOURCE_IMAGE_NAME}:${SOURCE_IMAGE_TAG}
+            paths:
+              - destinationDir: tmp
+                sourcePath: /opt/app-root/src/dist/admin-nrpti/.
+        type: Dockerfile
+      strategy:
+        dockerStrategy:
+          from:
+            kind: ImageStreamTag
+            name: ${RUNTIME_IMAGE_NAME}:${RUNTIME_IMAGE_TAG}
+        type: Docker
+      successfulBuildsHistoryLimit: 5
+      triggers:
+        - type: ImageChange
+          imageChange:
+        - type: ConfigChange
+parameters:
+  - description: A name used for all objects
+    displayName: Name
+    name: NAME
+    required: true
+    value: admin-nrced
```

Comment:
This should actually be nrpti

Reply:
`admin-nrced` is the name of the build, not the image.  The image name is still `nrpti` as discussed.  Changing this value will require changes to the `Jenkinsfile` and then changes to the builds on OCP3.